testing/testing_test.bzl

```diff
@@ -1,5 +1,5 @@
 """
-Macros for domain testing tests.
+Macros for testing tests.
```

Comment:
Yep. :) It is a bit awkward.

Reply:
Maybe "Macros for tests in the testing module." ?

Super small nit but this might also be something to update in the other `.bzl` files too, I noticed the docstrings have the same format and it was only after looking at the other modules that I could understand this sentence😄